source/_components/binary_sensor.netatmo.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: "Netatmo Binary Sensor"
+description: "Instructions how to integrate Netatmo binary sensor into Home Assistant."
+date: 2016-09-19 15:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: netatmo.png
+ha_category: Binary Sensor
+ha_release: 0.29
+---
+
+
+The `netatmo` binary sensor platform is consuming the information provided by a [Netatmo Welcome](https://www.netatmo.com) camera. This component allows you to get the latest event seen by the camera.
+
+To enable the Netatmo binary sensor, you first have to set up [netatmo](/components/netatmo/), this will use discovery to add you binary sensor, nothing else to do.
+
+If you want to select a specific sensor, set discovery to False for [netatmo](/components/netatmo/) and add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
```

Comment:
Minimize this example to it's bare limits. In general assume it's going to be cut and pasted.